#By defualt OpenCV looks IPP lib dir in $IPPROOT../lib
# In our setup it is at $IPPROOT/lib
# By B. Hajgato July 14th 2016
--- cmake/OpenCVFindIPP.cmake.orig	2015-12-18 16:02:16.000000000 +0100
+++ cmake/OpenCVFindIPP.cmake	2016-07-14 13:12:21.541285380 +0200
@@ -191,7 +191,7 @@
     _ipp_add_library(m)
 
     if(UNIX)
-      get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/../lib REALPATH)
+      get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/lib REALPATH)
       if(NOT EXISTS ${INTEL_COMPILER_LIBRARY_DIR})
         get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/../compiler/lib REALPATH)
       endif()
